### PR TITLE
Update to Go 1.19.6

### PR DESCRIPTION
**Pull request by jhchabran**
Update to 1.19.6 because it fixes a bunch of CVEs.

[_Created by Sourcegraph batch change `jhchabran/update-to-go-1.19.6`._](https://sourcegraph.sourcegraph.com/users/jhchabran/batch-changes/update-to-go-1.19.6)

Test Plan: CI

### CODE REVIEW

**Comment by jhchabran** [comment]
@coury-clark can't enable automerge without forcing it to wait for your review, so feel free to merge on approval!

**Review by coury-clark — APPROVED** [review]

**Pull request merged**
